src/_base.scss

```diff
@@ -187,30 +187,39 @@ h6 {
 
 
 /**
- * Base `.heading` styles.
+ * Headline. `.heading-headline` styles.
  */
 
-.heading {
-    color: $shell-g-color-text;
-    font-size: rem($shell-g-font-size);
-    font-weight: $shell-g-font-weight-normal;
+.heading-headline {
+    font-size: rem($shell-g-font-x-size-large);
+    font-weight: $shell-g-font-weight-bold;
+}
+
+/**
+ * Modifier: Large headline.
+ */
+
+.heading-headline--large {
+    font-size: rem($shell-g-font-size-2x-large);
+    font-weight: $shell-g-font-weight-bold;
 }
 
+
 /**
- * Modifiers: Main heading.
+ * Title. `.heading-title` styles.
  */
 
-.heading--main {
+.heading-title {
     font-size: rem($shell-g-font-size-large);
-    font-weight: $shell-g-font-weight-light;
 }
 
+
 /**
- * Modifiers: Section heading.
+ * Sub-title. `.heading-sub-title` styles.
  */
 
-.heading--section {
-    font-weight: bold;
+.heading-sub-title {
+    font-weight: $shell-g-font-weight-bold;
 }
 
 
```

Comment:
I can't remember if we discussed it already but do we need to bake in `margin-bottom`s for these?